app/views/applications/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<h1>Applicant Information</h1>
+
+<h3>Applicant's Name: </h3> <%= @application.name %>
+<h3>Applicant's Address: </h3> <%= @application.address %>
+<h3>Description of why I would make a good home: </h3> <%= @application.description %>
+
+<h3>Pets Applied for: </h3>
+<% @application.pets.each do |pet| %>
+<p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %> </p>
+<% end %> 
+<h3>Application Status: </h3> <%= @application.status %>
+
+<% if @application.status == "In Progress" %>
+  <h3> Add a Pet to this Application:</h3>
+  <%= form_with url: "/applications/#{@application.id}?pet_name=#{params[:pet_name]}&commit=Search", method: :get do %>
```

Comment:
If you're using a form, you shouldn't need to have the query parameters in this path (i.e. you shouldn't need to include `?pet_name=#{params[:pet_name]}&commit=Search` in the url on line 15 as they should be added automatically. Any field you put in a form will be put in the params object!

Reply:
Also, sometimes Turbo complains about forms that redirect back to the same path, so if you saw any weirdness, remember to do `data: {turbo: false}`!